pwiz_tools/Skyline/FileUI/ImportTransitionListColumnSelectDlg.cs

```diff
@@ -989,6 +1013,9 @@ private void ButtonOk_Click(object sender, EventArgs e)
 
         public void OkDialog()
         {
+            // Wait until the window is fully shown to do processing below
+            WaitForShown();
```

Comment:
Isn't this line of code guaranteed to cause a hang if we happen to get here and we haven't been shown yet? 
There is no way for event thread code to run while we are waiting, so I don't see how _windowShownEvent.Set() will ever manage to be called.